mathesar/reflection.py

```diff
@@ -93,9 +92,8 @@ def reflect_new_table_constraints(table):
 def reflect_db_objects():
     if not cache.get(DB_REFLECTION_KEY):
         reflect_databases()
-        for database_key in get_non_default_database_keys():
-            reflect_schemas_from_database(database_key)
+        for database in models.Database.current_objects.all():
+            reflect_schemas_from_database(database.name)
         for schema in models.Schema.current_objects.all():
             reflect_tables_from_schema(schema)
-        reflect_constraints_from_database(database_key)
```

Comment:
Please also replace the use of `database_key in this line.

Reply:
this is the path `mathesar\database\utils.py/get_data_key()`

Reply:
@arvind-1 Yes, you can delete it. Thanks!